pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -71,18 +90,38 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (ctr
 		return res, err
 	}
 
-	return ctrl.Result{}, q.runListRestorePoints(ctx, req)
+	dispatcher := q.makeDispatcher(q.Log, q.Vdb, q.PRunner, q.Password)
+	// Create a TLS secret for the NMA service
+	err := q.generateCertsForNMA(ctx)
```

Comment:
I don't understand why we are generating a cert here. We need to reuse the one the NMA already has present.